php/tests/wrapper_type_setters_test.php

```diff
@@ -225,4 +225,113 @@ public function constructorWithWrapperTypeDataProvider()
             [TestWrapperSetters::class, BytesValue::class, 'bytes_value', 'getBytesValue', "nine"],
         ];
     }
+
+    // /**
+    //  * @dataProvider invalidConstructorWithWrapperTypeDataProvider
+    //  * @expectedException \Exception
+    //  */
+    // public function testInvalidConstructorWithWrapperType($class, $wrapperField, $value)
+    // {
+    //     new $class([$wrapperField => $value]);
+    // }
+
+    // public function invalidConstructorWithWrapperTypeDataProvider()
+    // {
+    //     return [
+    //         [TestWrapperSetters::class, 'repeated_string_value', null],
+    //         // The below is an error case, but we can't test it as the RepeatedField
+    //         // class uses trigger_error in this case.
+    //         // [TestWrapperSetters::class, 'repeated_string_value', [null]],
+    //         [TestWrapperSetters::class, 'repeated_string_value', [new stdClass()]],
+    //         [TestWrapperSetters::class, 'map_string_value', null],
+    //         // The below is an error case, but we can't test it as the MapField
+    //         // class uses trigger_error in this case.
+    //         //[TestWrapperSetters::class, 'map_string_value', ['key' => null]],
+    //         [TestWrapperSetters::class, 'map_string_value', ['key' => new stdClass()]],
+    //     ];
+    // }
+
+    /**
+     * @dataProvider constructorWithRepeatedWrapperTypeDataProvider
+     */
+    public function testConstructorWithRepeatedWrapperType($class, $wrapperField, $getter, $value)
+    {
+        $actualInstance = new $class([$wrapperField => $value]);
+        foreach ($actualInstance->$getter() as $key => $actualWrapperValue) {
+            $actualInnerValue = $actualWrapperValue->getValue();
+            $expectedElement = $value[$key];
+            if (is_object($expectedElement) && is_a($expectedElement, '\Google\Protobuf\StringValue')) {
+                $expectedInnerValue = $expectedElement->getValue();
+            } else {
+                $expectedInnerValue = $expectedElement;
+            }
+            $this->assertEquals($expectedInnerValue, $actualInnerValue);
+        }
+    }
+
+    public function constructorWithRepeatedWrapperTypeDataProvider()
+    {
+        $sv7 = new StringValue(['value' => 'seven']);
+        $sv8 = new StringValue(['value' => 'eight']);
+
+        $testWrapperSetters = new TestWrapperSetters();
+        $testWrapperSetters->setRepeatedStringValue([$sv7, $sv8]);
+        $repeatedField = $testWrapperSetters->getRepeatedStringValue();
+
+        return [
+            [TestWrapperSetters::class, 'repeated_string_value', 'getRepeatedStringValue', []],
```

Comment:
Could you wrap the line?

Reply:
Shortened line by removing `TestWrapperSetters::class`